config/config.zkevm.prover.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
same as above, we need to configure the environment and the environment requires the prover.

Reply:
same here, use config file and/or env

Reply:
I don't know how to set the ENV var for the prover, so I moved the `sql` and `config` files to the `test/vectors/src/prover` to try to mimic what the protocol team is doing here: https://github.com/0xPolygonHermez/zkevm-prover/blob/main/testvectors/config.json

@ARR552 wdyt? is it better now?

Reply:
Ok, it's better keep this things isolated
I think it is better because we could face the situation where the prover configuration had changed and we didn't know it. I would keep things separated. Each config file in each repo to not mess things